src/codeu/chat/util/Encryptor.java

```diff
@@ -0,0 +1,141 @@
+/**
+ * Encryptor.java
+ * Provides various encryption-related methods.
+ */
+
+package codeu.chat.util;
+
+import javax.crypto.*;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.security.*;
+
+public class Encryptor {
+    private static final String SYMMETRIC_ALGORITHM = "AES";
+    private static final String ASYMMETRIC_ALGORITHM = "RSA";
+
+    public static SecretKey makeSymmetricKey() {
+        KeyGenerator keyMaker;
+
+        try {
+            keyMaker = KeyGenerator.getInstance(SYMMETRIC_ALGORITHM);
+        } catch(NoSuchAlgorithmException nsae) {return null;}
+
+        keyMaker.init(128);
+        return keyMaker.generateKey();
+    }
+
+    public static KeyPair makeAsymmetricKeyPair() {
+        KeyPairGenerator keyMaker;
+
+        try {
+            keyMaker = KeyPairGenerator.getInstance(ASYMMETRIC_ALGORITHM);
+        } catch(NoSuchAlgorithmException nsae) {return null;}
+
+        keyMaker.initialize(2048);
+        return keyMaker.generateKeyPair();
+    }
+
+    public static byte[] wrap(SecretKey keyToWrap, PublicKey publicKey) {
+        Cipher cipher;
+        try {
+            cipher = Cipher.getInstance(ASYMMETRIC_ALGORITHM);
+        }
+        catch(GeneralSecurityException gse) {return null;}
+
+        try {
+            cipher.init(Cipher.WRAP_MODE, publicKey);
+        }
+        catch (InvalidKeyException iee) {System.out.println("INVALID public key."); return null;}
+
+        try {
+            return cipher.wrap(keyToWrap);
+        }
+        catch (InvalidKeyException iee) {System.out.println("INVALID symmetric key.");}
```

Comment:
Please do not swallow exceptions, The caller will have no way to realize something went wrong.

Consider defining a custom exception class if you want the caller to be made aware of the error or at least exit the application if the error is non-recoverable.